src/pages/resultsView/mutation/MutationMapperStore.ts

```diff
@@ -154,7 +154,10 @@ export class MutationMapperStore {
         await: () => [
             this.mutationData
         ],
-        invoke: async () => fetchOncoKbData(this.sampleIdToTumorType, this.mutationData)
+        invoke: async () => fetchOncoKbData(this.sampleIdToTumorType, this.mutationData),
+        onError: (err: Error) => {
+            // fail silently, leave the error handling responsibility to the data consumer
```

Comment:
In case of an OncoKB fetch error, we are showing the error icon within the Annotation column now, instead of a pop up "Sorry something went wrong" error message